src/common/QuickActions/index.js

```diff
@@ -27,9 +28,11 @@ class QuickActions<TItem> extends React.Component<Props<TItem>, State> {
     isDeleteModalVisible: false,
   };
 
-  _hideDeleteModal = (): void => this.setState({ isDeleteModalVisible: false });
+  _hideDeleteModal = (): void =>
+    this.setState((): $Shape<State> => ({ isDeleteModalVisible: false }));
```

Comment:
Didn't you remove the lint requirement for types on functions like this?

Reply:
I removed but I still want to annotate as much as possible. Maybe you're right though, we don't need to annotate any standard functions.

Reply:
Yeah.. this doesn't help much and I hate seeing `$Shape` :)